test/CMakeLists.txt

```diff
@@ -13,40 +13,40 @@ include(../cmake/tools.cmake)
 
 # ---- Dependencies ----
 
-CPMAddPackage("gh:doctest/doctest@2.4.8")
+# XXX CPMAddPackage("gh:doctest/doctest@2.4.8")
+CPMAddPackage("gh:iboB/doctest-lib@2.4.8")
 CPMAddPackage("gh:TheLartians/Format.cmake@1.7.3")
 
-if(TEST_INSTALLED_VERSION)
-  find_package(Greeter 1.0 REQUIRED)
+if(TEST_INSTALLED_VERSION) # FIXME: if(PROJECT_IS_TOP_LEVEL)
```

Comment:
The main question: is `test` as `top level project` dependent of installed version by default?
In original version **not** and use for `test_coverage`!